src/components/Callout/Callout.Props.ts

```diff
@@ -81,3 +95,8 @@ export interface ILink {
    */
   url: string;
 }
+
+export enum CalloutBackgroundColor {
+  white,
+  blue
```

Comment:
Ok, this is a different mental model than what I'm used to, but I can see where you're going with it.

I'm used to saying "TeachingBubble IS A callout that has custom defined content inside of it", and you're saying "TeachingBubble USES callout but imposes specific overrides and styles upon it".